hiplan/base/board.py

```diff
@@ -24,5 +25,23 @@ def add_members(self, member: Member = None):
     def serialize(self):
         return {
             'name': self.name,
-            'records': [record.serialize() for record in self.records]
+            'records': [record.serialize() for record in self.records],
+            'description': self.description
+            #'members': [member.serialize() for member in self.members]
         }
+
+    @staticmethod
+    def deserialize(dict):
+        board = Board(name=dict['name'], records=None, description=dict['description'], members=None)
+        board.records = []
```

Comment:
Asta este default behaviour in constructor. Check [here](https://github.com/mariaaliu/HiPlan/pull/6/files#diff-84e744afe43b103e3f42d89192ebf4bfa7867fc650e720c78ec2c24b9b76ef34R9)